src/api-client.ts

```diff
@@ -37,7 +40,7 @@ export const getApiClient = function (
   return new githubUtils.GitHub(
     githubUtils.getOctokitOptions(auth, {
       baseUrl: getApiUrl(apiDetails.url),
-      userAgent: "CodeQL Action",
+      userAgent: `CodeQL ${getMode()} ${pkg.version}`,
```

Comment:
I believe this is the convention and follows the HTTP spec.
```suggestion
      userAgent: `CodeQL-${getMode()}/${pkg.version}`,
```